### DISCUSSION TITLE
빌드잇에서 가져온 맵에서 MeshCollider 이슈

### DISCUSSION THREAD

**Discussion by lhj5709** [naverz/zepeto-studio-kor#375] [Issue Report (Legacy)]
안녕하세요

빌드잇에서 가져온 맵의 오브젝트에서 BoxCollider로 구성된 오브젝트는 물리효과(충돌)가 적용되는데  MeshCollider형태로 넘어온 오브젝트는 적용이 안되고 겹쳐지네요

혹시 해결할 수 있는 방법이 있을까요 ?

---------------------------------------------
[https://studio.zepeto.me/guides/loading-buildit-maps](url)

추가)) 혹시 위 링크의 유의사항에 메쉬컴바인을 제공하지 않는다는 내용이 이 문제일까요 ?

**Comment by gusriel** [comment]
혹시 inspector에 Mesh Collider에 Convex 체크 해제 해보셨나요
